### ISSUE TITLE
Accessibility. point.id.indexOf is not a function

### ISSUE THREAD

**Issue by dknight** [highcharts/highcharts#17401]
#### Expected behaviour
In accessibility module's medthod `getPointXDescription()` there is a chunk
```js
point.id.indexOf('highcharts-') < 0
```
but point.id might be a number, Number class doesn't have method `indexOf` This causes the error `TypeError: point.id.indexOf is not a function`

#### Actual behaviour
```js
String(point.id).indexOf('highcharts-') < 0
```
#### Product version
Highcharts, 10.1.0

#### Affected browser(s)
All

**Comment by pawellysy** [comment]
Hello @dknight, thanks for reporting a bug!
Here is the simplified demo: https://jsfiddle.net/BlackLabel/9b1p5vkn/178/
